fuzz/main.c

```diff
@@ -58,11 +63,34 @@ usage(const char *exec_name)
 		"  -v:		   increase verbosity\n"
 		"  -l:		   list tests by name\n"
 		"  -f:		   halt after first failure\n"
-		"  -n <STRING>:  only run tests STRING in the name\n",
+		"  -n <STRING>:  only run tests STRING in the name\n"
+		"  -s <SEED>:	  fuzz from the given seed\n",
 		exec_name);
 	exit(1);
 }
 
+static theft_seed
+parse_seed(const char *s)
+{
+	unsigned long u;
+	char *e;
+
+	errno = 0;
+
+	u = strtoul(s, &e, 16);
```

Comment:
Does this need to be `strtoull`? The seed is a typedef of `uint64_t`, so it could get truncated as a long.